src/edu/csus/ecs/pc2/exports/ccs/ResultsFile.java

```diff
@@ -55,8 +55,8 @@ public void setFinalizeData(FinalizeData finalizeData) {
      * @return
      * @throws IllegalContestState
```

Comment:
Since the changes below removed the `throws IllegalContestState`, the above line should be deleted.